problems/kattis/quadrant/java/jasonBengtson/Quadrant.java

```diff
@@ -0,0 +1,41 @@
+import java.io.*;
+import java.util.*;
+
+public class Main {
+
+    static void Quad(double x, double y) {
+        if(x > 0 && y > 0)
+            System.out.println("1");
+        else if(x < 0 && y > 0)
+            System.out.println("2");
+        else if(x < 0 && y < 0)
+            System.out.println("3");
+        else if(x > 0 && y < 0)
+            System.out.println("4");
+        else
+            System.out.println("Incorrect input");
+
+    }
+
+

```

Comment:
Ok, Jason I didn't download this because you had a fork problem. But I did take your code from here and copy it into a java file and run it. It runs as it should +10